docs/reference/glossary.md

```diff
@@ -0,0 +1,15 @@
+| Term    | Description                          |
```

Comment:
I'd suggest adding a page title and hiding the table of contents as this will otherwise be blank and use space.

```suggestion
---
hide:
  - toc
---

# Glossary

| Term    | Description                          |
```